internal/k8sCommon/k8sclient/pod.go

```diff
@@ -41,9 +42,11 @@ func (c *podClient) NamespaceToRunningPodNum() map[string]int {
 	if !c.inited {
 		c.Init()
 	}
-	if c.store.Refreshed() {
-		c.refresh()
-	}
+	//if c.store.Refreshed() {
+	//	log.Printf("I! store refresh %v", c.store.refreshed)
+	//	c.refresh()
+	//}
+	c.refresh()
```

Comment:
Is this a breaking code change in the k8s code?

Reply:
Same question for me 👍

Reply:
@ZhenyuTan-amz can you test if this still works after removing the comments. I only tested in this state.

Reply:
I don't think removing the comments has any consequence on the execution. I was just asking why it changed because I don't recall seeing a `refresh()` function for the pod client.

Reply:
Tested and it works with the code commented out. I'm not concerning removing comment would affect anything.